vendor/vendor.json

```diff
@@ -155,92 +155,98 @@
 		{
 			"checksumSHA1": "I7uTxTP05oqFbpuwtaPCl0fqElE=",
 			"path": "github.com/keybase/client/go/auth",
-			"revision": "433eb48ff4c25a822386faabcc40fd304269db33",
-			"revisionTime": "2017-10-11T18:56:40Z"
+			"revision": "9fd1a7236d9b727eaa242d8e41496e3b79388785",
```

Comment:
I don't see this revision in `client/master`.  Can you update it to a real, merged commit?

Reply:
Oops, done! Didn't we have an automated checker for this somewhere?